src/form/Select/Select.tsx

```diff
@@ -639,7 +645,7 @@ export const Select: FC<Partial<SelectProps>> = ({
 
   const onPasteHandler = useCallback(
     (e: React.ClipboardEvent<HTMLInputElement>) => {
-      if (createable && multiple) {
+      if (createOnPaste && multiple) {
```

Comment:
Does it select if its already a 'created' option?

Reply:
yes

Reply:
if creatable = false but I paste in a option this condition is invalid. I think it should be `selectOnPaste` then when combined w/ `creatable` would create them or of not just select them.